bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -196,6 +195,7 @@ private func dummy() {
     _ = NSLocalizedString("Line Height", comment: "")
     _ = NSLocalizedString("Link inserted", comment: "")
     _ = NSLocalizedString("Link Rel", comment: "")
+    _ = NSLocalizedString("Link Settings", comment: "")
```

Comment:
This string is no longer used in the web version after [this change](https://github.com/WordPress/gutenberg/pull/51712/files#diff-4b02749cf17631881372540e1ff4bed69a8ce72fc214e7208660ae1c1abffab0L32).